plugins/DuskVerb/tools/tuner/calibrate_octave_t60.py

```
@@ -0,0 +1,133 @@
+    # readings (reported 9/9 while a clean build measured 5/9). Disable the cache
+    # so every iteration links the actual edited map. (memory: ccache staleness.)
+    env = dict(os.environ, CCACHE_DISABLE="1")
+    r = subprocess.run(["cmake","--build","build","--target","DuskVerb_VST3","duskverb_render",
+                        f"-j{os.cpu_count()}"], cwd=REPO, capture_output=True, text=True, env=env)
+    if r.returncode != 0:
+        print(r.stdout[-3000:]); print(r.stderr[-3000:]); sys.exit("BUILD FAILED")
+
+
+def render(name):
+    for f in glob.glob(f"{OUTD}/*_noiseburst.wav"):
+        try: os.remove(f)
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)